### ISSUE TITLE
spatial_lr_scale

### ISSUE THREAD

**Issue by Jaswar** [ingra14m/Deformable-3D-Gaussians#87]
Hi, thanks for sharing this work!

I noticed that the `spatial_lr_scale` is hard-coded to the value 5:
[https://github.com/ingra14m/Deformable-3D-Gaussians/blob/ac423728df780ee57a82025a6d7c4c7312fdf9bb/scene/gaussian_model.py#L88](https://github.com/ingra14m/Deformable-3D-Gaussians/blob/ac423728df780ee57a82025a6d7c4c7312fdf9bb/scene/gaussian_model.py#L88)

This is opposed to the original 3DGS definition, where the scale is computed automatically and passed as a parameter:
[https://github.com/graphdeco-inria/gaussian-splatting/blob/54c035f7834b564019656c3e3fcc3646292f727d/scene/gaussian_model.py#L150](https://github.com/graphdeco-inria/gaussian-splatting/blob/54c035f7834b564019656c3e3fcc3646292f727d/scene/gaussian_model.py#L150)

Changing the `spatial_lr_scale` to use the passed parameter, degrades the performance significantly. I tried to figure out how the value `5` was chosen, but did not find any information about it. Would it be possible for you to offer an explanation for this setting?

**Comment by JianxiangL** [comment]
Do you have any further understanding of this? I'm quite curious as well.

**Comment by miya9756** [comment]
I think that is a reasonable magic number for these dynamic datasets. 
In the D-NeRF dataset, a single object is quite easy to fit so using higher learning rate can improve performance. 
For NeRF-DS dataset, the actual scene scale is larger than the one calculated by cameras. So strictly following the 3DGS's training setup will cause some issues.
